tests/helpers.py

```diff
@@ -1,28 +1,25 @@
 from pathlib import Path
+import time
 from urllib.parse import urlparse
-import boto3
+
 from nucleus import DatasetItem, BoxPrediction
 
 PRESIGN_EXPIRY_SECONDS = 60 * 60 * 24 * 2  # 2 days
 
 TEST_MODEL_NAME = "[PyTest] Test Model"
-TEST_MODEL_REFERENCE = "[PyTest] Test Model Reference"
 TEST_MODEL_RUN = "[PyTest] Test Model Run"
 TEST_DATASET_NAME = "[PyTest] Test Dataset"
 TEST_SLICE_NAME = "[PyTest] Test Slice"
 
-TEST_MODEL_NAME = "[PyTest] Test Model Name"
-TEST_MODEL_REFERENCE = "[PyTest] Test Model Reference"
-TEST_MODEL_RUN = "[PyTest] Test Model Run Reference"
-TEST_DATASET_NAME = "[PyTest] Test Dataset"
-TEST_SLICE_NAME = "[PyTest] Test Slice"
+
 TEST_IMG_URLS = [
-    "s3://scaleapi-cust-lidar/Hesai/raw_data/2019-5-11/hesai_data_1557540003/undistorted/front_camera/1557540143.650423lf.jpg",
-    "s3://scaleapi-cust-lidar/Hesai/raw_data/2019-5-11/hesai_data_1557540003/undistorted/back_camera/1557540143.600352lf.jpg",
-    "s3://scaleapi-cust-lidar/Hesai/raw_data/2019-5-11/hesai_data_1557540003/undistorted/right_camera/1557540143.681730lf.jpg",
-    "s3://scaleapi-cust-lidar/Hesai/raw_data/2019-5-11/hesai_data_1557540003/undistorted/front_left_camera/1557540143.639619lf.jpg",
-    "s3://scaleapi-cust-lidar/Hesai/raw_data/2019-5-11/hesai_data_1557540003/undistorted/front_right_camera/1557540143.661212lf.jpg",
+    "http://farm1.staticflickr.com/107/309278012_7a1f67deaa_z.jpg",
```

Comment:
Why are we not using s3?

Reply:
to avoid dependency on AWS creds in CircleCI